src/Common/tests/System/IO/Compression/ZipTestHelper.cs

```diff
@@ -133,6 +137,14 @@ public static void StreamsEqual(Stream ast, Stream bst, int blocksToRead)
             IsZipSameAsDir(s, directory, mode, requireExplicit, checkTimes);
         }
 
+        public static byte[] NormalizeLineEndings(byte[] str)
+        {
+            string rep = Text.Encoding.Default.GetString(str);
+            rep = rep.Replace("\r\n", "\n");
+            rep = rep.Replace("\n", "\r\n");
```

Comment:
Is this correct? You replace all `\r\n` line characters with `\n` line characters and afterwards replace them back from `\n` to `\r\n`. Why is the first instruction needed? (line 143)

Reply:
I wanted to normalize line endings to `\r\n` to match what most of the test files have, but if I just did  `("\n", "\r\n")` then it would end up with some `\r\r\n`